packages/clerk-js/src/ui/signIn/SignIn.tsx

```diff
@@ -31,8 +31,8 @@ function SignInRoutes(): JSX.Element {
       </Route>
       <Route path='sso-callback'>
         <SSOCallback
-          afterSignInUrl={signInContext.afterSignInUrl}
-          afterSignUpUrl={signInContext.afterSignUpUrl}
+          afterSignInUrl={signInContext.afterSignInUrl || signInContext.redirectUrl}
+          afterSignUpUrl={signInContext.afterSignUpUrl || signInContext.redirectUrl}
```

Comment:
❓ Can we use use the `redirectUrl` prop directly? Eg:

```ts
        <SSOCallback
          afterSignUpUrl={signUpContext.afterSignUpUrl}
          afterSignInUrl={signUpContext.afterSignInUrl}
          redirectUrl={signUpContext.redirectUrl}
          secondFactorUrl={signUpContext.signInUrl + '#/factor-two'}
        />
```

Reply:
Of course we can. Thanks for suggesting it.